kubernetes/client/exceptions.py

```diff
@@ -83,7 +83,8 @@ def __init__(self, msg, path_to_item=None):
 
 class ApiException(OpenApiException):
 
-    def __init__(self, status=None, reason=None, http_resp=None):
+    def __init__(self, status=None, reason=None, http_resp=None,
+                 request_method=None, request_url=None, request_headers=None):
```

Comment:
This file is generated by [openapi-generator](https://github.com/OpenAPITools/openapi-generator). Please make the change in the upstream

Reply:
Thanks. I did the pull request and have good chances to get it merged

https://github.com/OpenAPITools/openapi-generator/pull/13215